source/platforms/BitBucketServer.ts

```diff
@@ -223,10 +225,15 @@ export class BitBucketServer implements Platform {
         }
       }
     } else {
-      await this.createComment(newComment)
+      issue = await this.createComment(newComment)
     }
 
-    return true
+    // We want to make a URL like:
+    // https://bitbucket.org/atlassian/jiraconnect-ios/pull-requests/4/crash-groups/diff#comment-426
+    // But I don't see
```

Comment:
Did you forget to add the rest of the comment? 😄

Reply:
lol yep, took a while to figure out how to make the link